_candidates_en/BasGeerdink_2.md

```diff
@@ -0,0 +1,24 @@
+---
+name: Bas Geerdink
+title: "Analytics ad ING - the Why, What, and How of building a data-driven financial enterprise"
+length: 40
+audience: Intermediate
+language: English
+twitter: bgeerdink
+github: geerdink
+icon: https://pbs.twimg.com/profile_images/911184285830348800/q0O6D7iA_400x400.jpg
+organization: Technology Lead at ING
+tags:
+  - Big Data / Fast Data
+  - Best Practices
+  - Data Science / Machine Learning
+  - DevOps
+  - Software Design and Architecture
+suggestions:
+ - Engineers, managers and architects who want to learn how a big financial enterprise is doing big data and fast data
+---
+ING is a Data-Driven Experimental Enterprise, which is heavily investing in big data, analytics and streaming processing. As in many other enterprises, we deal with a large variety of data sources. The amount of data which must be handled goes beyond the computing performance of single machines, and vertical scalability is hardly an option. For this specific reason we are moving towards a scenario where machines are grouped into clusters and data is handled by distributed processing systems.
+
+Important building blocks in our journey are the state-of-the-art Data Lake, and the Streaming Data Platform. The data lake, built around Hadoop and IBM PDA, replaces several enterprise data warehouses and is the central repository for all types of data, supporting various types of queries for our stakeholders' demands. Data is also being handled more often than not as streams and we are working with Flink and Kafka to provide faster, more reactive and up-to-date user experiences and journeys. Finally, machine learning is aiding traditional SQL to provide better insight when it comes to operational excellence, business processes, marketing and security applications.
+
+In this talk, Bas will explain why ING is becoming more and more data-driven, and how we do it. He will share use cases, architecture, and technology choices.
```

Comment:
@geerdink I think this is well over the 600 character limit of CPF abstract.
Could you please post a shorter version here, or send us a PR on this file?